devimint/src/external.rs

```diff
@@ -77,16 +77,18 @@ impl Bitcoind {
     }
 
     pub async fn mine_blocks(&self, block_num: u64) -> Result<()> {
-        info!(block_num, "Mining bitcoin blocks");
+        info!(?block_num, "Mining bitcoin blocks");
```

Comment:
Is there a difference?

Reply:
> The ? sigil is shorthand that specifies a field should be recorded using its [fmt::Debug](https://doc.rust-lang.org/nightly/core/fmt/trait.Debug.html) implementation

But in this case it doesn't make any difference